ios/LayoutReanimation/REASharedTransitionManager.m

```diff
@@ -392,10 +422,15 @@ - (void)screenRemovedFromStack:(UIView *)screen
     [self restoreViewsVisibility];
   } else {
     // removed stack
-    [self clearConfigForStack:stack];
+    [self maybeClearConfigForStack:stack isInteractive:[self isInteractiveScreenChange:screen]];
   }
 }
 
+- (bool)isInteractiveScreenChange:(UIView *)screen
+{
+  return [[[screen.reactViewController valueForKey:@"transitionCoordinator"] valueForKey:@"interactive"] boolValue];
```

Comment:
I think we should be able to access this using public API. IIRC there is a way to extract this information via UINavController